### PR TITLE
Feature/register other org sprint 31.5 EXUI_MO_3.0.63

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EUI-8811
https://tools.hmcts.net/jira/browse/EUI-8801
https://tools.hmcts.net/jira/browse/EUI-8777
https://tools.hmcts.net/jira/browse/EUI-8672
https://tools.hmcts.net/jira/browse/EUI-8756
https://tools.hmcts.net/jira/browse/EUI-8758
https://tools.hmcts.net/jira/browse/EUI-8797
https://tools.hmcts.net/jira/browse/EUI-8730
https://tools.hmcts.net/jira/browse/EUI-8786
https://tools.hmcts.net/jira/browse/EUI-8821
https://tools.hmcts.net/jira/browse/EUI-8731
https://tools.hmcts.net/jira/browse/EUI-8814
https://tools.hmcts.net/jira/browse/EUI-8873
https://tools.hmcts.net/jira/browse/EUI-8729
https://tools.hmcts.net/jira/browse/EUI-8869
https://tools.hmcts.net/jira/browse/EUI-8943
https://tools.hmcts.net/jira/browse/EUI-8951
https://tools.hmcts.net/jira/browse/EUI-8926
https://tools.hmcts.net/jira/browse/EUI-8945
https://tools.hmcts.net/jira/browse/EUI-8927
https://tools.hmcts.net/jira/browse/EUI-8937
https://tools.hmcts.net/jira/browse/EUI-8928
https://tools.hmcts.net/jira/browse/EUI-8923
https://tools.hmcts.net/jira/browse/EUI-8831
https://tools.hmcts.net/jira/browse/EUI-8929
https://tools.hmcts.net/jira/browse/EUI-8921
https://tools.hmcts.net/jira/browse/EUI-8841

### Change description ###

Register Organisation Feature behind a feature Toggle.

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [ ] Does this PR introduce a breaking change

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

File: src/app/utils/app-utils.spec.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { LovRefDataModel } from 'src/shared/models/lovRefData.model';
```
Comment (connorpgpmcelroy) [inline src/app/utils/app-utils.spec.ts:1]: src

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with minor comment

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: src/app/utils/app-utils.spec.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/utils/app-utils.spec.ts]: Done

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by Daniel-Lam — APPROVED** [review]
@udaydenduluri33 - consolidation for Sprint 31.5 approved

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Pull request merged**